hail/python/requirements.txt

```diff
@@ -15,6 +15,6 @@ parsimonious<0.9
 plotly>=5.5.0,<5.11
 protobuf==3.20.2
 PyJWT
-pyspark>=3.1.1,<3.2.0
+pyspark==3.3.0
```

Comment:
Seems reasonable to do `pyspark>=3.3,<3.4`?

Reply:
I guess I was distrusting of pyspark and wanted the exact same version that there is in dataproc, but maybe that is unfounded.